### DISCUSSION TITLE
Windows 10 search in windows 11.

### DISCUSSION THREAD

**Discussion by randomidiot2887** [valinet/ExplorerPatcher#3700] [Ideas]
I really want a windows 10 style search menu for windows 11. my idea is to make it look like windows 10, but be better than windows 11 with integration with everything search, which is 10 times faster than windows search. can someone consider adding this to here? i searched and couldnt find any other ways to get this.

**Comment by pyrates999** [comment]
that sounds like an entirely new application and is beyond what EP is for, sorry.

**Comment by milestprower92** [comment]
The Windows 10 search menu can't be restored on Windows 11 22H2+

**Reply by randomidiot2887** [reply]
why?
if so. ok.

**Reply by jsdg1242-ctrl** [reply]
can it be remade and put there
if it cant then ok

**Comment by randomidiot2887** [comment]
ok. i closed this discussion then
